cnxarchive/sql/get-users-by-ids.sql

```diff
@@ -0,0 +1,17 @@
+-- ###
+-- ###
+SELECT row_to_json(user_row) 
+FROM   (SELECT username                           AS id, 
+               first_name                         AS firstname, 
+               last_name                          AS surname, 
+               full_name                          AS fullname, 
+               title                              AS title, 
+               (SELECT Array_agg(value) 
+                FROM   contact_infos AS ci 
+                WHERE  ci.user_id = u.id 
+                       AND TYPE = 'EmailAddress') AS emails, 
```

Comment:
I think `TYPE = 'EmailAddress'` should be lowercase `type = 'EmailAddress'` if `type` is a column name.